packages/fast-tooling-react/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@microsoft/fast-tooling-react",
   "description": "FAST Tooling React",
```

Comment:
Would it be possible to improve the description with some indication of what the tooling does?  I imagine a sentence or two may be better.

Reply:
Sure, I can do that in a different PR.